src/components/WebexInMeeting/WebexInMeeting.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexLocalMedia, WebexRemoteMedia} from '..';
+
+import './WebexInMeeting.css';
+
+/**
+ * Webex InMeeting component displays the remote stream plus
+ * the local stream at the bottom right corner.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexInMeeting({meetingID}) {
+  return (
+    <div className="in-meeting">
+      <React.Fragment>
```

Comment:
If you're enclosing the media components in a div, the `Fragment` is unnecessary, please remove it